src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Kindly make sure that in this step of the project, it's required to practice the class-based component, and this means to convert the `function` keyword to `Class`, please let's work on that! 😄 🙏